dataplanes/vpp/cmd/dataplanetestclient/dataplane-test-client.go

```diff
@@ -120,13 +122,13 @@ type dataplaneClientTest struct {
 func main() {
 	flag.Parse()
 
-	if *srcPodName == "" || *dstPodName == "" {
-		logrus.Fatal("Both source and destination PODs' name must be specified, exitting...")
-	}
-	k8s, err := buildClient()
-	if err != nil {
-		logrus.Fatal("Failed to build kubernetes client, exitting...")
-	}
+	//if *srcPodName == "" || *dstPodName == "" {
+	//	logrus.Fatal("Both source and destination PODs' name must be specified, exitting...")
+	//}
+	//k8s, err := buildClient()
+	//if err != nil {
+	//	logrus.Fatal("Failed to build kubernetes client, exitting...")
+	//}
```

Comment:
If these lines are going to be permanently commented out, should we then remove them?